<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="langComboBox">
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="themeComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Сохранить как...</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="actionJson"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Открыть как...</string>
     </property>
     <addaction name="actionTXT"/>
     <addaction name="actionJson_2"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_3">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionJson">
   <property name="text">
    <string>Json</string>
   </property>
  </action>
  <action name="actionTXT">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionJson_2">
   <property name="text">
    <string>Json</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_5</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_6</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_7</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_8</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_9</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_10</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
